assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -8,19 +8,17 @@
     <div class="row col-12 m-0 p-0 p-md-2 py-2 text-size-3">
         <div class="{% if can_delete_all or can_delete_some %} col-10 col-sm-11 {% else %} col-12 {% endif %}">
             <div class="row pr-2">
-                <div class="col-12 col-md-6 p-0 show-md">Name</div>
-                <div class=" text-center col-12 col-md-6 pr-3">
-                    <div class="row dining-help-row">
-                        <div class="col-3 p-0">Shopped</div>
-                        <div class="col-3 p-0">Cooked</div>
-                        <div class="col-3 p-0">Cleaned</div>
-                        <div class="col-3 p-0">Paid</div>
-                    </div>
+                <div class="col-12 col-md-6 p-0 show-md"><b>Name</b></div>
+                <div class=" text-center col-12 col-md-6 d-none d-md-flex">
+                        <div class="col-3 p-0"><b>Shopped</b></div>
+                        <div class="col-3 p-0"><b>Cooked</b></div>
```

Comment:
(Indentation klopt niet)

Reply:
Indentation klopt, regel 11 bevat het begin en einde van de div

Reply:
Het zat 'm in een extra tab in regel 13-16 maar dat komt wel een keer vanzelf goed met automatische formatters (bijv. `ctrl` `alt` `L` in PyCharm)